### DISCUSSION TITLE
Please make tagada

### DISCUSSION THREAD

**Discussion by ghost** [OpenRCT2/OpenRCT2#18798] [Feature requests]
Please add tagada to Roller Coaster Tycoon 2

**Comment by spacek531** [comment]
This looks like a custom object for RCT3.

**Reply by ghost** [reply]
I hope that this will be added to rct2 in a rct2 feel

**Comment by karst** [comment]
I'd love to see more flat rides in general, a pendulum ride, a swing/starflyer, tea cups, booster, amongst others

**Reply by spacek531** [reply]
The good news is that in theory new flat rides aren't difficult. The bad news is nobody's actually made one and nobody has made it their priority.

**Reply by 733737** [reply]
actually there are lots of custom flat rides, but many are glitchy or very old

**Reply by spacek531** [reply]
custom objects are not what I mean by new flat rides

**Comment by frutiemax** [comment]
I'd like to make ride types become objects first. I've tried to do a giant ferris wheel a while ago but it ended with too much bloated code. The drawing was working but the operation code was too much a mess.

**Reply by karst** [reply]
If you'd make a new base element, say 7x3, couldn't you then make flat rides using that base using the current ride type objects?

**Reply by frutiemax** [reply]
The base drawing is the easy part it's the peep drawing and structure, also how it's animated but also how peeps embark and disembark

**Reply by ocalhoun6** [reply]
> If you'd make a new base element, say 7x3, couldn't you then make flat rides using that base using the current ride type objects?

It's *relatively* easy to make a new flat ride, as long as that ride can be accomplished simply by re-skinning an existing flat ride. But if your flat ride is an unusual shape or needs the peeps to behave differently as they getting on and off the ride, then it will likely require a lot of extra code.

(And by relatively easy ... it's still quite an undertaking. Requires a lot of brand-new, custom-made sprites. But it *can* be done. For example, a custom ride like [this](https://ae.rctspace.com/tutorial/Ch04-SuperLooper.jpg) was created by reskinning a swinging inverter ship.)
